### ISSUE TITLE
Possibility of adding a coordinate system?

### ISSUE THREAD

**Issue by bytrangle** [mathieudutour/svg-path-visualizer#25]
Hello Mathieu. Thank you very much for this fantastic tool.

This is not an issue report, but what do you about adding a coordinate system to help visualize the paths even better? I can help with that.

It's alright if that doesn't align with your goal for this tool. It is still a game-changing tool for me regardless.

**Comment by mathieudutour** [comment]
Yeah that's a good idea. Some work has been started here: https://github.com/mathieudutour/svg-path-visualizer/pull/1. Happy to review a PR
